### PR TITLE
E_CLASSROOM-382 [Improvement][Student] Display only one spinner when retrieving quizzes and related quizzes

### PR DESCRIPTION
**Pull request by CaneteJoash**
### Issue Link
https://framgiaph.backlog.com/view/E_CLASSROOM-382
### Definition of Done
- [x] Only one spinner for all the quizzes and related quizzes should be displayed
- [x] No separate spinner for each

### Commands to Run
N/A

#### Main note
- I removed the spinner in Quiz list under subcategory and in the Quiz List so that it will have only one spinner.
- But I implement the spinner in the quiz result
#### Pages Affected
- http://localhost:3003/categories/3/sub
- http://localhost:3003/categories/3/quizzes
- http://localhost:3003/categories/3/quizzes/2/questions

### Scenarios/ Test Cases
- [ ] Only one spinner for all the quizzes and related quizzes should be displayed
- [ ] No separate spinner for each
#### User Interface
N/A
#### User Experience 
N/A
#### Functionality
N/A
### Screenshots
image
image
image

### CODE REVIEW

**Comment by catampongan** [comment]
## Approved PR #198 
- [x] All data load at once with only one spinner
----------------
task-382
task-382-2

--------------------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

File: src/pages/student/categories/CategoryDetail/components/QuizzesCard/index.js [hunk 1]
```diff
@@ -59,60 +58,51 @@ const QuizzesCard = ({ category_id, quiz }) => {
...
 
   return (
-    <>
-      {questions === null ? (
-        <div className={style.loading}>
-          <Spinner animation="border" role="status"></Spinner>
-          <span className={style.loadingWord}>Loading</span>
-        </div>
+    <Col className={style.cardContainer}>
+      {getTotalTimeLimit() === 0 ? (
```
Comment (karlolee50) [inline src/pages/student/categories/CategoryDetail/components/QuizzesCard/index.js]:
```suggestion
      {!getTotalTimeLimit() ? (
```

File: src/pages/student/quizzes/QuizList/components/QuestionGrid/index.js [hunk 2]
```diff
@@ -59,59 +58,44 @@ const QuestionGrid = ({ quiz }) => {
...
+          </div>
+          <div className={style.ResultScore} style={{ fontWeight: 'bold' }}>
+            <p>Latest Score</p>
+            <p>
+              {getLatestScore() >= 0 ? getLatestScore() : 0}/{questions?.length}
+            </p>
+          </div>
+        </div>
+      </Card.Body>
+      {quiz.answerCount === 0 || (
```
Comment (karlolee50) [inline src/pages/student/quizzes/QuizList/components/QuestionGrid/index.js:95]:
```suggestion
      {!quiz.answerCount || (
```

File: src/pages/student/quizzes/QuizResult/index.js [hunk 3]
```diff
@@ -153,12 +154,18 @@ const QuizResult = ({ score, total, quizId, categoryId }) => {
                 </center>
               </div>
             ) : (
-              <div className={style.relatedQuizzes}>
-                {quizRelated?.map((relatedQuiz, idx) => {
-                  return <Recent relatedQuiz={relatedQuiz} key={idx} />;
-                })}
-              </div>
-            )}
+              quizRelated === null ? (
```
Comment (karlolee50) [inline src/pages/student/quizzes/QuizResult/index.js]:
```suggestion
              !quizRelated ? (
```

File: src/pages/student/quizzes/QuizResult/index.module.scss [hunk 4]
```diff
@@ -167,3 +167,12 @@
...
 }
+
+.spinner{
+  display: flex;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  margin-top: 50px;
+  gap: 10px
+}
```
Comment (karlolee50) [inline src/pages/student/quizzes/QuizResult/index.module.scss]:
```suggestion
}

```

add extra line

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 5 minutes
Comment/s: 4

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**